### ISSUE TITLE
Welcome to Daytona

### ISSUE THREAD

**Issue by scripting** [scripting/drummerSupport#128]
Daytona is search for outlines, and that includes:

1. Drummer docs.

2. Scripting News.

3. Your public Drummer outlines. 

Read all about it [here](http://docserver.scripting.com/drummer/search.opml), and post questions, kudos and love here. ❤️

**Comment by scripting** [comment]
to @akaKenSmith @am1t @papascott @mistersugar @sabre23t -- continuing the previous discussion...

I decided I couldn't trade off linking and style against highlighting, i had to have both, so that's what i'm working on this morning. it seems to be possible.

**Comment by scripting** [comment]
Bing! (I'm glad you guys are getting to watch the development process. That's how you keep your sanity while building something complex. You tell yourself you can't do this, so the problem is simplified, but eventually once it's working it becomes approachable. Now I hope this *really* works. It could prove that I missed an edge case. It happens.)

![image](https://user-images.githubusercontent.com/1686843/145605932-7ebf326f-af18-4c35-b05d-9a894a688b56.png)

**Comment by mistersugar** [comment]
I wasn't much help during the Daytona testing, but I will report that
yesterday when I was wanting to find something I'd previously written, I
reflexively went to Daytona, searched, and found. Only then did I realize
what I'd done and what you've made. So thanks a million.

---
*Anton Zuiker*
919.923.9491

Blogging again at zuiker.com

**Comment by mistersugar** [comment]
Reporting a quirky behavior but no damage:

Yesterday I was using Daytona. Then I noticed last night that Drummer had two tabs open and both were my blog.opml. One tab had the lightning bolt. I was very confused and worried that something had been overwritten, but determined that I had not lost anything. 

I just used Daytona and clicked the eye icon and was taken to Drummer where the same thing happened - a second tab opened with my blog.opml. I closed the second tab, went back to Daytona, and clicked the eye icon again, and this time a new browser window opened with Drummer and one tab for blog.opml and the other browser window switched to Drummer docs, as expected.

**Comment by scripting** [comment]
When you write these, please edit out the fear. It makes it really hard for me to focus -- so emotional. 

It's a style that I really dislike. That's a bug report about a bug report. ;-)

**Comment by akaKenSmith** [comment]
<img width="376" alt="Screen Shot 2021-12-10 at 12 06 22 PM" src="https://user-images.githubusercontent.com/4759847/145613789-12ca733c-9819-4fcd-9f0c-4a1a156eff65.png">

Aren't the two tabs both with the blog explained in this way? The first is the editable blog file already open in Drummer. The second is an instant outline copy sent to Drummer by Daytona.

**Comment by papascott** [comment]
I was just able to reproduce this... in Drummer it's possible to have a public outline opened twice... once with the "Open…" menu for editing, then again with "Open URL…" as an Instant Outline.

**Comment by scripting** [comment]
yup that's an area of drummer that needs work. i took a look at it a couple of weeks ago and decided to do daytona. it was too exciting an idea not  to see through.

**Comment by fmcpherson** [comment]
Daytona is only returning search results contained in nodes with type attribute of outline. I've been testing Daytona against a OPML file containing books I ready in 2020: http://drummer.scripting.com/frankm/2020BooksRead.opml 

Originally none of the nodes had a type attribute and searches on author names did not return search results even though the file is public. I then added a type of outline to a node and the corresponding search of an author's name succeeded.

The addition of Daytona is incentive to me to move most of the files I have been maintaining in Little Outliner to Drummer, but as most of those don't have type attributes set it appears I need to write up a script... unless in the future the search will ignore type values. I'll wait to see on the decision.

**Comment by fmcpherson** [comment]
This may be a nit, but I find how History and the Search input field interact, or actually don't interact confusing. Here is what do and see:

1. Enter a search term and click Search. Observe the search results.
2. Click History. Observe the prior search term was added to the list. Click another, entered prior search term
3. Observe the results of the prior search display but the term you entered in step 1 above still displays in the input field but underneath that you see the results summary, eg. 1 docs match "Search term from history menu". 

The overall display is confusing to me. What I expect to happen is that either when I select a search from History that search term is populared in to the search input filed on the page or the input field is blanked out so there isn't a difference in appearance between what I see in the search input box and the actual search results.

**Comment by fmcpherson** [comment]
I think it would be useful that if a search results that show results from multiple docs that the doc file name or title be displayed somewhere in the result. If you added the doc file name or title to the right of the top level date nodes and then added Collapse All I think the result would appear more meaningful, right now a collapse all action would display a list of dates.

**Comment by scripting** [comment]
@fmcpherson -- not sure if this is what you were reporting about the History menu, but I broke it in the change I made this morning. It's fixed now.

**Comment by scripting** [comment]
@fmcpherson -- re only indexing typed nodes, that's the documented behavior. please, read the docs. thanks. ;-)

**Comment by fmcpherson** [comment]
@scripting I didn't see the docs until you published the search. Sorry. However, I think not including nodes that don't have a type attribute in search is creating an inconsistency with how you are handling such nodes in Old School. I referring to the situation prior when Old School would only publish nodes of type outline and you removed that constraint.

**Comment by scripting** [comment]
@fmcpherson -- just read the docs ok.

**Comment by scripting** [comment]
BTW the docs are linked into this post, it's the ONLY LINK in the post. 

This is an incredible feature -- never been done before, and we needed it.

I declare a moratorium on nits for at least the next few days. I'm going to keep rolling and I refuse to become a support slave to people who won't even read the fucking manual.
